### PR TITLE
Handling for generic validation error

### PR DESCRIPTION
**Pull request by mdrillin**
Addresses Issue #278 
- added handling for GenericValidationResults in CreateConnectorComponent
- some numeric inputs require positive int - added POS-INT type and sets a different min value for those on FormInputComponent
- fix issue in CreateConnectorComponent, was checking DataOptions instead of RuntimeOptions

### CODE REVIEW

File: ui/packages/ui/src/app/pages/createConnector/CreateConnectorComponent.tsx [hunk 1]
```diff
@@ -529,6 +537,7 @@ export const CreateConnectorComponent: React.FunctionComponent<ICreateConnectorC
                 }
               }
             }
+            alert(JSON.stringify(result));
```
Comment (indraraj) [inline ui/packages/ui/src/app/pages/createConnector/CreateConnectorComponent.tsx:540]: Need to remove the alert.

**Review by indraraj — CHANGES_REQUESTED** [review]
UI seems to be breaking on the RuntimeOptions step for Postgres.

**Comment by mdrillin** [comment]
Amended the commit to fix issues
- removed alert (oops)
- fixed problem in Utils which caused RuntimeStep error
- I also removed alert on DataOptions and RuntimeOptions step, since I was getting duplicate success messages as shown below on those steps:

DuplicateMsg

**Review by indraraj — APPROVED** [review]

**Comment by indraraj** [comment]
Tested locally, Looks good.

**Pull request merged**